tests/synthdnn/README.md

```diff
@@ -0,0 +1,28 @@
+# Synthdnn
+
+Synthdnn is a suite of scripts for collecting and analyzing oneDNN performance
+across a randomly generated data. The general architecture is intended to follow
+a data pipeline composed of synthetic problem generation, data collection, and
+data analysis. The `synthdnn.py` script provides a command line interface to
+these tools. Sample Usage:
+
+
+Data Generation:
+```
+python3 synthdnn.py <primitive> [sampling controls] -b <batch_file>
+```
+Data Collection: Not yet implemented.
+
+Data Generation and Collection:
+```
+python3 synthdnn.py <primitive> [sampling controls] --engine=<engine> --collect <data_kind> <benchdnn_file>
+```
+
+Report Generation: Not yet implemented, intended usage:
+```
+python3 synthdnn.py <primtive> --engine=gpu --collect perf <benchdnn_file> > data.log
+python3 synthdnn.py report --report-kind <kind>  data.log
+
+```
+
+See `synthdnn.py -h` for additional details.
```

Comment:
I recommend to move all Python stuff to scripts/ folder where verbose_converter lives.